website/docs/getting-started/setup.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 0
+---
+
+# Setup
+
+Functionless requires a few prerequisite setup steps to ensure your environment
+is ready to build and deploy the code you write as infrastructure to AWS.
+
+## Need Help?
+
+Setting up an AWS Account to use with Functionless can be daunting if you don't
+have familiarity with AWS. If you get stuck, [please don't hesitate to reach out
+in discord.][getting-help]
+
+## Install AWS CLI
+
+This step is not required, but strongly recommended for ensuring your
+environment is properly configured.
+
+[Install the AWS CLI.][install-aws-cli]
+
+## Configure AWS Access
```

Comment:
I haven't yet looked into it - but I'd like to learn about aws sso. Is that relevant here or no?

Reply:
AWS SSO is pretty involved. I would consider it outside the scope of getting started, but something we could cover in something like an advanced topics section.